support/sql/releases/sqlserver-2019/cumulativeupdate24.md

```diff
@@ -31,6 +32,11 @@ SQL Server 2019 CU14 introduced a [fix to address wrong results in parallel plan
 
 Microsoft is working on a fix for this issue and it will be available in a future CU.
 
+###Issue 2:
```

Comment:
### Read-scale availability group not displayed in dm_hadr_database_replica_cluster_states